### ISSUE TITLE
Problem uploading attachments

### ISSUE THREAD

**Issue by johandse** [ON-IT/Visma.Net#93]
I try to add a pdf attachment to an invoice, but the attachment becomes corrupt. In the following code I expect Lenght1 and Length2 to have the same value, but they don´t... Do I use this feature wrong, or is it a problem with the library?

```csharp
var File1 = new System.IO.FileStream(@"C:\MyFile.pdf", System.IO.FileMode.Open);
var Length1 = File1.Length;
File1.Close();
var Att1 =  vismaNet.CustomerInvoice.AddAttachmentToInvoice("000146", new System.IO.FileStream(@"C:\MyFile.pdf", System.IO.FileMode.Open), "MyFile.pdf").Result;
var File2 = vismaNet.GetAttachment(Att1).Result;
var Length2 = File2.Length;
```

**Comment by omelhus** [comment]
I'll look into this.

**Comment by omelhus** [comment]
This is som weird. I'm getting the same, but if I try with a plain text file all is good.

I'll let you know when I understand some more.

**Comment by omelhus** [comment]
There's definitely something funky when uploading PDF-files. I'm creating a post on the forum.

**Comment by omelhus** [comment]
[Forum post](https://www.visma.com/software/support/portal/forum/showthread.php?11890-Upload-attachment-on-CustomerInvoice-corrupts-PDF-files&p=24800)

**Comment by omelhus** [comment]
Turns out I was doing uploads wrong. Fixed in dev and a new release coming up.

```csharp
Task.Run(async () =>
{
    var visma = new VismaNet(clientId, token);
    var file = File.ReadAllBytes(localPath);
    var attachment =
        await visma.CustomerInvoice.AddAttachmentToInvoice(invoiceNumber, file,
            Path.GetFileName(localPath));
    using (var src = await visma.GetAttachment(attachment))
    using (var output = File.Create(outputTest))
    {
        await src.CopyToAsync(output);
        Console.WriteLine($"Uploaded file was {file.Length.ToString().PadLeft(12)} bytes");
        Console.WriteLine($"Downloaded file was {src.Length.ToString().PadLeft(10)} bytes");
    }
});
```

**Comment by omelhus** [comment]
This was fixed in 3.3.2.

**Comment by johandse** [comment]
Can confirm this is working in 3.3.2, thanks!
